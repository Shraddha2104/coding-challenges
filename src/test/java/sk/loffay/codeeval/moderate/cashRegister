314.79;314.79
259.20;305.20
81;81
216.91;216.91
30.15;30.15
122;122
100;100
63.00;89.00
63.00;89.35
208;219
63;89.35
183;183
16;16
202;176
339;403.22
313.96;241.45
15.94;16.00
398.50;400.25
296.17;297
75;75
