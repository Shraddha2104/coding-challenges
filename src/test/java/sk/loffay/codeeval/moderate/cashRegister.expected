ZERO
TWENTY,TWENTY,FIVE,ONE
ZERO
ZERO
ZERO
ZERO
ZERO
TWENTY,FIVE,ONE
TWENTY,FIVE,ONE,QUARTER,DIME
TEN,ONE
TWENTY,FIVE,ONE,QUARTER,DIME
ZERO
ZERO
ERROR
FIFTY,TEN,TWO,TWO,DIME,DIME,PENNY,PENNY
ERROR
NICKEL,PENNY
ONE,HALF DOLLAR,QUARTER
HALF DOLLAR,QUARTER,NICKEL,PENNY,PENNY,PENNY
ZERO
